internal/output/markdowntable.go

```diff
@@ -9,11 +10,15 @@ import (
 )
 
 // PrintMarkdownTableResults prints the osv scan results into a human friendly table.
+// The output includes a 'Fixed Version' column, showing the version where the vulnerability is fixed, if available.
```

Comment:
We don't need to specifically mention a fixed version here. If you want to emphasize that this contains all columns from the table output, we can state that the Markdown table uses the same format as the table output and includes the same fields.